tests/unit/lms/product/plugin/course_copy_test.py

```diff
@@ -118,3 +133,7 @@ def test_set_mapped_file_id(self, helper):
     @pytest.fixture
     def helper(self):
         return CourseCopyFilesHelper()
+
+    @pytest.fixture
+    def store_new_course_files(self):
+        return create_autospec(lambda _: None)  # pragma: nocover
```

Comment:
No cover, the lambada only exists to create the spec, but it never gets called itself.